vcstools/general_utils.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import logging
+from astropy.time import Time
+from time import strptime, strftime
+from astropy.utils import iers
+
+logger = logging.getLogger(__name__)
+
+def sfreq(freqs):
+
+    if len(freqs) != 24:
+        print("There are not 24 coarse chans defined for this obs. Got: %s" % freqs)
+        return
+
+    #freqs.sort()   # It should already be sorted, but just in case...[SET] Commenting this out because sort() is ironically putting 2-digit channels out of order
+    lowchans = [f for f in freqs if f <= 128]
+    highchans = [f for f in freqs if f > 128]
+
+    highchans.reverse()
+    freqs = lowchans + highchans
+
+    return freqs
+
+
+def is_number(s):
```

Comment:
why would we ever need a function to check if something is an int?

Reply:
It's used in obs_max_min to find files with obsids in their names.
```
def obs_max_min(obsid, meta=None):
    """
    Small function to query the database and return the times of the first and last file
    """

    # Make a list of gps times excluding non-numbers from list
    times = [f[11:21] for f in get_files(obsid) if is_number(f[11:21])]
    obs_start = int(min(times))
    obs_end = int(max(times))
    return obs_start, obs_end
```

Reply:
oh i see, because the obsid is in string form we need to check if it can be converted to an int